tests/format/markdown/with-plugins/format.test.js

```diff
@@ -0,0 +1,6 @@
+import prettierPluginToml from "../../../config/prettier-plugins/prettier-plugin-dummy-toml/index.js";
+
+runFormatTest(import.meta, ["markdown"], {
+  proseWrap: "always",
+  plugins: [prettierPluginToml],
+});
```

Comment:
Maybe better move to `tests/misc/front-matter/`, so we can test all languages supports it there.